### DISCUSSION TITLE
v0.10.10 - Detect unknown areas/devices/entities in automations & scripts

### DISCUSSION THREAD

**Discussion by frenck** [frenck/spook#14] [Announcements]
# 👻👻👻👻👻👻👻👻👻👻

- Add service: `input_number.min`
- Add service: `input_number.max`
- Add service: `number.min`
- Add service: `number.max`
- Fix/correct number increment/decrement services to handle unit conversions
- Add FAQ section to the README
- Added a whole bunch of repairs, that can detect the use of unknown areas, devices, and services used in your automation and scripts 👻

![image](https://user-images.githubusercontent.com/195327/221708987-4413fa5b-5524-443e-9acc-c7414a1327f3.png)

# 👻👻👻👻👻👻👻👻👻👻

## New Contributors

* @PiotrMachowski made their first contribution in https://github.com/frenck/spook/pull/12

**Full Changelog**: https://github.com/frenck/spook/compare/v0.6.780...v0.10.10

<hr /><em>This discussion was created from the release <a href='https://github.com/frenck/spook/releases/tag/v0.10.10'>Boooooooooooooo!</a>.</em>

**Comment by timmo001** [comment]
Actually super useful as for why some of my automations havent been triggering. Thanks for this!

<img width="541" alt="image" src="https://user-images.githubusercontent.com/28114703/221899994-84f4da29-b3c0-4eba-b830-805d65611975.png">
